main.c

```diff
@@ -31,7 +30,8 @@ int main(int argc, char **argv) {
     char c;      /* INPUT CHARACTER / BLOCK NUMBER */
     char bks[3]; /* BLOCKS GIVEN TO USER */
     char bka[3]; /* FLAGS OF BLOCKS AVAILABLE ; TODO -> BITARRAY */
-    char b;      /* INDEX OF USER CHOSEN BLOCK */
+    char b;      /* INDEX OF USER CHOSEN BLOCK (3 = NO SELECTION) */
+    char oldb;   /* PREVIOUSY SELECTED BLOCK, REQUIRED FOR NEXT/PREV BLOCK */
```

Comment:
well, only required if you don't want the position to reset when switching blocks with no other block available